ENG-2545 - Password fields now are not always shown (#3676)
FusionAuth/fusionauth-site: astro/src/content/docs/get-started/core-concepts/users.mdx

These are the default fields that are available when creating a user. You can add additional fields by creating custom forms and fields. See the [Editing User Data In the admin UI](/docs/lifecycle/register-users/advanced-registration-forms#editing-user-data-in-the-admin-ui) documentation for more information.

<Aside>

>   The password related fields are only available if the following are configured for the tenant:

Reviewer: ```suggestion
  The password-related fields are only available if the following are configured for the tenant:
```

      Then only the setup password message will be sent to the user. Setting up the password using the message sent during this user create operation will verify the user’s email address or phone number if it is not already verified.
    </Aside>
    <Aside>

>       <InlineField>Send password setup to</InlineField> is only available if the following tenant settings are:

Author: NOTE that the absence of template settings does NOT prevent this from showing up on the form, it just prevents user creation from working (the User API will reject the request when it validates).

Not sure if that distinction should be made here.

>       <InlineField>Send password setup to</InlineField> is only available if the following tenant settings are:

Reviewer 2: Ending with "tenant settings are" feels like it's missing part of the sentence to me. Does this work?
```suggestion
      <InlineField>Send password setup to</InlineField> is only available if the following are configured for the tenant:
```

Author: Yes. I think when I was playing with the wording I forgot to go back and change this. suggestion accepted.

>       <InlineField>Send password setup to</InlineField> is only available if the following are configured for the tenant:

Author: Agreed. Split the difference, see what you think.

      * [Setup password option](/docs/get-started/core-concepts/tenants#template-settings) - template settings must be set up

>       * [Password enabled](/docs/get-started/core-concepts/tenants#password) - set to `true`

Author: Did not say "only applies to version 1.99.9" here because thought that might be too busy. Might be best to explain that on the tenant docs, when https://linear.app/fusionauth/issue/ENG-2547/document-tenant-password-enabled-option is done

    </Aside>
  </APIField>
  <APIField name="Password" required>
    The password of the user.

>     <Aside>

Author: Consolidated into a single aside at the top of the page.

      This field is only displayed if:

>       * <InlineField>Send password setup to</InlineField> is not enabled or is set to `Do not send`

Author: Earlier references to `<InlineField>Send email to set up password</InlineField>` were missed. Fixed that.

      * [Password enabled](/docs/get-started/core-concepts/tenants#password) - set to `true`
    </Aside>
  </APIField>
  <APIField name="Confirm password" required>
    Password confirmation field. You must enter the same value in both <InlineField>Password</InlineField> and <InlineField>Confirm password</InlineField>.

    <Aside>
      This field is only displayed if:
      * <InlineField>Send password setup to</InlineField> is not enabled or is set to `Do not send`

>       * [Password enabled](/docs/get-started/core-concepts/tenants#password) - set to `true`

Reviewer: ```suggestion
      * [Password enabled](/docs/get-started/core-concepts/tenants#password) is set to `true`
```
